components/FlexibleHeader/src/MDCFlexibleHeaderView.h

```diff
@@ -20,6 +20,27 @@ typedef void (^MDCFlexibleHeaderChangeContentInsetsBlock)(void);
 typedef void (^MDCFlexibleHeaderShadowIntensityChangeBlock)(CALayer *_Nonnull shadowLayer,
                                                             CGFloat intensity);
 
+/** This determines how we adjust the height to account for the Safe Area insets. */
+typedef NS_ENUM(NSInteger, MDCFlexibleHeaderHeightSafeAreaAdjustment) {
```

Comment:
lets make it clear what it adjusts:
MDCFlexibleHeaderHeightSafeAreaAdjustsMinMaxHeight